extensions/azurecore/src/account-provider/azureAccountProvider.ts

```diff
@@ -172,8 +198,18 @@ export class AzureAccountProvider implements azdata.AccountProvider, vscode.Disp
 			account.isStale = true;
 			Logger.error(`_getAccountSecurityToken: Authentication method not found for account ${account.displayInfo.displayName}`);
 			throw Error('Failed to get authentication method, please remove and re-add the account');
+		}
+	}
+
+	encryptAccountSecurityToken(token: Token): Promise<string> {
+		return this._encryptAccountSecurityToken(token);
+	}
 
+	private async _encryptAccountSecurityToken(token: Token): Promise<string> {
```

Comment:
nit, but we don't usually have private functions start with _. At the very least we should be consistent, but isValidToken below doesn't follow that same pattern.

Reply:
I get this is trying to follow "internal" method naming pattern for exposed APIs - as exists for `_getSecurityToken` and `_getAccountSecurityToken` and so on..